x-pack/solutions/security/plugins/cloud_security_posture/public/pages/benchmarks/benchmarks.test.tsx

```diff
@@ -38,7 +38,7 @@ describe('<Benchmarks />', () => {
           cspm: { status: 'indexed' },
           kspm: { status: 'indexed' },
           indicesDetails: [
-            { index: 'logs-cloud_security_posture.findings_latest-default', status: 'not-empty' },
+            { index: 'logs-cloud_security_posture.misconfiguration_latest', status: 'not-empty' },
```

Comment:
same here `security_solution-cloud_security_posture.misconfiguration_latest` pattern, but really let's make sure these tests are doing what we think they should be doing.